loadtest/config.go

```diff
@@ -81,7 +75,7 @@ type UsersConfiguration struct {
 type Config struct {
 	ConnectionConfiguration     ConnectionConfiguration
 	UserControllerConfiguration UserControllerConfiguration
-	InstanceConfiguration       InstanceConfiguration
+	InstanceConfiguration       gencontroller.Config
```

Comment:
I have to admit I don't like this particularly but the alternative of having an identical structure and copy over the data wasn't perfect either.

Reply:
`GenController` config would then be coupled with a higher level construct. I tend to prefer this direction when possible (higher level struct including lower level struct).

Reply:
Generally, that would be okay. But the main question I had while reading the code is why `gencontroller`? Because `gencontroller` is one such controller from a whole lot of controllers. What is so special about that it needs to be included in the loadtest package?

My reasoning was if we do the other way round, gencontroller is just taking a top-level struct which is free to be used by any low-level packages. Other controllers may or may not use that, and that's fine.

Reply:
> I mean that all the flag options in the init command aren't actually used by the init command. They are just shown because init is a sub-command of loadtest and are inherited from the parent command's flags. My point was to investigate whether those flags are meant to be inherited by any other commands than the main command, and if yes, then replace PersistentFlags with Flags.

Understood. Yes, some flags are clearly not meant to be part of the `init` subcommand. I'll remove them.

Reply:
Yes, the confusion is reasonable because it looks odd indeed. Maybe it's fine to have `InstanceConfiguration` be a structure that mirrors (at least for now) `gencontroller.Config`.
It just means copy over the values in `cmd/ltagent/init.go`.

Reply:
That works. A little duplication is better than bringing odd package dependencies.